x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/custom_assets_extension.tsx

```diff
@@ -8,13 +8,31 @@
 import React from 'react';
 import { type CustomAssetsAccordionProps, CustomAssetsAccordion } from '@kbn/fleet-plugin/public';
 import { i18n } from '@kbn/i18n';
+import { useLocation } from 'react-router-dom';
+import { EuiSpacer } from '@elastic/eui';
+import { VULN_MGMT_POLICY_TEMPLATE } from '../../../common/constants';
 import { useKibana } from '../../common/hooks/use_kibana';
 import { benchmarksNavigation, cloudPosturePages } from '../../common/navigation/constants';
 
 const SECURITY_APP_NAME = 'securitySolutionUI';
 
 export const CspCustomAssetsExtension = () => {
   const { application } = useKibana().services;
+  const { search } = useLocation();
+  const isCNVM = search.includes(VULN_MGMT_POLICY_TEMPLATE);
```

Comment:
Can you change `useLocation` to `useParams`? It's more robust since we can specify the query param to look for. 

Suggestion:

```js 
const integration = useParams<{ integration: CloudSecurityPolicyTemplate | undefined }>().integration;
```